sourcemash/api/feeds.py

```diff
@@ -24,25 +24,28 @@ def output(self, key, feed):
 class getUnreadCount(fields.Raw):
     def output(self, key, feed):
         total_item_count = Item.query.filter_by(feed_id=feed.id).count() 
-        
-        if not current_user.is_authenticated():
-            return total_item_count 
-        
         read_item_count = UserItem.query.filter_by(user=current_user, feed_id=feed.id, unread=False).count()
-
         return total_item_count - read_item_count
 
 feed_fields = {
     'id': fields.Integer,
     'title': fields.String,
     'url': fields.String,
+    'subscribed': isSubscribed
+}
+
+feed_fields_detailed = {
     'description': fields.String,
     'image_url': fields.String,
-    'last_updated': fields.DateTime,
+    'last_updated': fields.DateTime
+}
+feed_fields_detailed.update(feed_fields)
+
+feed_fields_status = {
     'item_count': getItemCount,
-    'unread_count': getUnreadCount,
-    'subscribed': isSubscribed
+    'unread_count': getUnreadCount
 }
+feed_fields_status.update(feed_fields)
```

Comment:
Since we "extend" from feed_fields, could we make this more readable by having feed_fields updated with feed_status_fields?

i.e.
```python
feed_status_fields = dict(feed_fields, **feed_status_fields)
```